extensions/apps/settings/src/components/pages/notifications-browser-option.tsx

```diff
@@ -85,7 +85,7 @@ const BrowserNotificationsOption: React.FC = () => {
   const handleEnableBrowserNotifications = async () => {
     setLoading(true);
     try {
-      await sdk.services.common.notification.listenToNotificationEvents();
+      await sdk.services.common.notification.enableBrowserNotifications();
```

Comment:
Changed this so only toggle browser setting. Listening should be initiated from navbar where we react to new notifications with bell icon